.github/workflows/codespell.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
I wonder though why this works did not run, may be some policy?

Reply:
> I wonder though why this works did not run, may be some policy?

Yes. There's a button I see below saying "Approve and run [workflows]". Probably because they can run code.